src/dateTimePicker.test.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+import DateTimePicker from './DateTimePicker';
+import CalendarComponent from './CalendarComponent';
+import TimeInput from './TimeInput';
+
+fdescribe('DateTimePicker', function() {
+
+	let dateTimeComponent,
+		calendarComponent,
+		timeComponent;
+		// dateTimeLocalComponent;
+
+	const year = 2018,
+		month = 0,
+		day = 1,
+		dateStr = `${year}-0${month+1}-0${day}`,
+		hours = 23,
+		minutes = 11,
+		datetime = new Date(Date.UTC(year, month, day, hours, minutes));
+
+	// datetime.setHours(hours, minutes);
+
+	beforeEach(() => {
+		dateTimeComponent = TestUtils.renderIntoDocument(
+			<DateTimePicker name='start_time'
+				value={datetime}
+				forceCalendar />);
+		calendarComponent = TestUtils.findRenderedComponentWithType(dateTimeComponent, CalendarComponent);
+		timeComponent = TestUtils.findRenderedComponentWithType(dateTimeComponent, TimeInput);
```

Comment:
`timeComponent` is only used in one test - no need to define it in `beforeEach`

`calendarComponent` is only used in one test, and that test makes its own value assignment, so the `beforeEach` assignment is redundant